packages/cryptoassets/src/currencies.js

```diff
@@ -2698,6 +2726,36 @@ const cryptocurrenciesById: { [name: string]: CryptoCurrency } = {
       },
     ],
   },
+  crypto_org_croeseid: {
+    type: "CryptoCurrency",
+    id: "crypto_org_croeseid",
+    coinType: 1,
+    name: "Crypto.org Croeseid Coin",
+    managerAppName: "Crypto.org Croeseid Chain",
```

Comment:
I don't see this application in my manager. Is it an advanced app?

Reply:
Croeseid is the testnet for Crypto.org Chain.

I dont think there is any app in the manager for it. But I though there was also Bitcoin Test or Ethereum Ropsten which does not have an app?

Should I just remove it ? (I am actually not sure if it is supported)

Reply:
managerAppName must be a real app name. It sounds like it should be Crypto.org Chain, if you confirm you use that same app to also use testnet.
The same way ethereum ropsten currency have defined Ethereum for the app

Reply:
Ok I see. I think it should be fine with "Crypto.org Chain" but I am not in the phase where I am able to test.
Nevertheless, I will change the value.